docs/site/BelongsTo-relation.md

```diff
@@ -75,18 +97,45 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
-The definition of the `belongsTo` relation is inferred by using the `@belongsTo`
-decorator. The decorator takes in a function resolving the target model class
-constructor and designates the relation type. It also calls `property()` to
-ensure that the decorated property is correctly defined.
+The standard naming convention for the foreign key property in the source model
+is `target model name` + `Id` (for example, Order.customerId).
+
+- If the foreign key property name in the source model has to be customized,
+  then the `name` attribute in the relation definition (the target model name)
```

Comment:
Same as above, the `name` here is relation name. Users can name the relation based on their preference. For instance, they can name it `orderCustomer` instead of `customer` in the `OrderRelations`. 

If the foreign key property name in the source model has to be customized, the relation name has to be specified in  the `name` attribute of the relation definition. In addition, if you have a the corresponding `hasMany` or `hasOne` relation in the `target` model, e.g a `Customer` has many `Order`s, the `keyTo` field of that corresponding property needs to be specify. Check the `#Relation Metadata` in [`hasMany`](https://loopback.io/doc/en/lb4/HasMany-relation.html) and [`hasOne`](https://loopback.io/doc/en/lb4/HasOne-relation.html) for more details.
(Should either add an example of `hasMany` or a link here.)

Reply:
agreed